lib/matplotlib/widgets.py

```diff
@@ -1232,6 +1494,433 @@ def funchspace(self, val):
             self.targetfig.canvas.draw()
 
 
+class AxesTool(Widget):
+    """
+    A tool to adjust the params of a `matplotlib.figure.Figure`.
+    """
+    
+    def __init__(self, targetfig, toolfig):
+        """
+        Parameters
+        ----------
+        targetfig : `.Figure`
+            The figure instance to adjust.
+        toolfig : `.Figure`
+            The figure instance to embed the changes into.
+        """
+        
+        self.targetfig = targetfig
+        self.toolfig = toolfig
+        self.toolfig.subplots_adjust(left=0.2, right=0.85)
+        axes = self.targetfig.get_axes()
+        self.ax, = axes
```

Comment:
This fails if there are multiple subplots (Qt currently uses a first combobox to select the axes to edit, in that case).